MBL-2351: Add feature flag for filtering by % raised

amy-at-kickstarter opened the pull request:
# 📲 What

Add a feature flag for phase 4 search filters, filtering by % raised.

scottkicks reviewed (approved):
👍 
I wonder if there are feature flags we can clean up now?

amy-at-kickstarter commented:
> 👍 I wonder if there are feature flags we can clean up now?

I cleaned up the previous search flag, but it would be cool if we cleaned up the late pledges one...
